src/v/utils/format.h

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2020 Redpanda Data, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+
+#pragma once
+#include <fmt/format.h>
+
+// Workaround for arm64 coroutine crash. See
+// https://groups.google.com/g/seastar-dev/c/Nt6LizrwE9U for details.
+template<typename S, typename... Args>
+static __attribute__((noinline)) std::string
+format(const S& format_str, Args&&... args) {
```

Comment:
nice. do you think if this is to be at the global level we should name it something more unique to avoid clashes?

Reply:
vformat to keep it in line with vlog and vassset